docs/reference/api/pieces.md

```diff
@@ -230,11 +230,43 @@ The PATCH request body may use MongoDB-style operators. For example, you may use
 
 The successful PATCH request returns the complete patched document. See the [piece document response example](#piece-document-response-example) below for a sample response body. On error an appropriate HTTP status code is returned.
```

Comment:
Is this a link to something immediately beneath this text? "piece document response example"

Reply:
Yeah, that's further down the page.